src/ZoneServer/Skills/Handlers/Swordsman/Highlander/Crown.cs

```diff
@@ -0,0 +1,104 @@
+    /// </summary>
+    [SkillHandler(SkillId.Highlander_Crown)]
+    public class Highlander_Crown : IGroundSkillHandler
+    {
+        private const float DefPenetrationPerLevel = 0.01f;
+        private const float BuffRemoveChancePerLevel = 33f;
+
+        /// <summary>
+        /// Handles skill, damaging targets.
+        /// </summary>
+        /// <param name="skill"></param>
+        /// <param name="caster"></param>
+        /// <param name="originPos"></param>
+        /// <param name="farPos"></param>
+        public void Handle(Skill skill, ICombatEntity caster, Position originPos, Position farPos, ICombatEntity target)
+        {
+            if (!caster.TrySpendSp(skill))
+            {
+                caster.ServerMessage(Localization.Get("Not enough SP."));
+                return;
+            }
+
+            skill.IncreaseOverheat();
+            caster.SetAttackState(true);
+
+            var splashParam = skill.GetSplashParameters(caster, originPos, farPos, length: 65, width: 25, angle: 0);
+            var splashArea = skill.GetSplashArea(SplashType.Square, splashParam);
+
+            Send.ZC_SKILL_READY(caster, skill, originPos, farPos);
+            Send.ZC_SKILL_MELEE_GROUND(caster, skill, farPos, null);
+
+            this.Attack(skill, caster, splashArea);
+        }
+
+        /// <summary>
+        /// Executes the actual attack after a delay.
+        /// </summary>
+        /// <param name="skill"></param>
+        /// <param name="caster"></param>
+        /// <param name="splashArea"></param>
+        private async void Attack(Skill skill, ICombatEntity caster, ISplashArea splashArea)
+        {
+            var hitDelay = TimeSpan.FromMilliseconds(300);
+            var damageDelay = TimeSpan.FromMilliseconds(50);
+            var skillHitDelay = TimeSpan.Zero;
+
+            await Task.Delay(hitDelay);
+
+            var targets = caster.Map.GetAttackableEntitiesIn(caster, splashArea);
+            var hits = new List<SkillHitInfo>();
+
+            foreach (var target in targets.LimitBySDR(caster, skill))
+            {
+                var modifier = SkillModifier.MultiHit(2);
+                modifier.DefensePenetrationRate = DefPenetrationPerLevel * skill.Level;
+
+                var skillHitResult = SCR_SkillHit(caster, target, skill, modifier);
+                target.TakeDamage(skillHitResult.Damage, caster);
+
+                var skillHit = new SkillHitInfo(caster, target, skill, skillHitResult, damageDelay, skillHitDelay);
+                skillHit.HitEffect = HitEffect.Impact;
+                hits.Add(skillHit);
+
+                var debuffTime = 5f + 0.5f * skill.Level;
+
+                // In earlier versions, this skill instead inflicted Common_Shock
+                target.StartBuff(BuffId.Crown_Armor_Debuff, skill.Level, 0, TimeSpan.FromSeconds(debuffTime), caster);
+
+                // Also need to potentially remove a buff from the target
```

Comment:
A well intentioned comment, but the block of code after it distracts from it, and once you oriented yourself and come to this comment, you probably already figured out what the code does.

Reply:
This was a holdover from before that functionality was implemented, so I've removed it.